### PR TITLE
fix #54: Allow getting a ConnectedAccount via email

### LINKED ISSUE #54: Add support to get account via email
```
      // TODO: Add support to get account via email
```

https://github.com/seamapi/javascript/blob/e9d712ac23a2b505a7a7369406508331ccfe59cc/src/cli/commands/connected-accounts.ts#L21

### PR DESCRIPTION
**Pull request by xplato**
This also added a test for the `ConnectedAccount` code because it doesn't seem to have been present before. This actually only tests the new email logic (and NOT getting a ConnectedAccount by `connected_account_id` because the property `seed.devices.augustLock.connectedAccountId` was `undefined` for some reason.

Any idea where else I could get that ID to write the proper test?

### CODE REVIEW

**Comment by xplato** [comment]
These GH tests are failing because of the new `manufacturer` property we just added.

File: src/types/models.ts [hunk 1]
```diff
@@ -160,8 +160,13 @@ export type AccessCode = OngoingAccessCode | TimeBoundAccessCode
...
   created_at: string
-  user_identifier: string
+  user_identifier: UserIdentifier
   account_type: Provider
+  errors: string[]
+}
+
+export interface UserIdentifier {
+  email: string
 }
```
Comment (codetheweb) [inline src/types/models.ts:170]: 🐧 minor style thing but this should probably go before `ConnectedAccount` since the latter depends on it

**Review by codetheweb — APPROVED** [review]
> Any idea where else I could get that ID to write the proper test?

You'll probably have to update `tests/fixtures/workers/helpers/add-fake-august-devices.ts` and/or `tests/fixtures/workers/helpers/add-fake-schlage-devices.ts`, I think the mapping for the connected account ID is wrong there.

File: src/types/models.ts (hunk 1 above)
Comment (xplato) [reply src/types/models.ts:170]: Good point! I will adjust it.

**Pull request merged**